draft-ietf-scitt-architecture.md

```diff
@@ -447,8 +443,8 @@ In these cases, distinct Issuers (`iss`) might agree to use the `sub` CWT Claim
 
 Transparency Service:
 
-: an entity that maintains and extends the Append-only Log, and endorses its state.
-A Transparency Service can be a complex system, requiring the Transparency Service to provide many security guarantees about its Append-only Log.
+: an entity that maintains and extends the Verifiable Data Structure, and endorses its state.
```

Comment:
```suggestion
: an entity that maintains and extends the Append-only Log, a Verifiable Data Structure that captures, and endorses it.
```

I think we still intend to have the TS maintain the log itself, as well as the VDS.

Reply:
Not sure about this, lets discuss

Reply:
We really need to decide if we're going to define the Append-only Log/ Verifiable Data Structure as two distinct things.

Reply:
deferring to #352